flake.nix

```diff
@@ -62,7 +62,7 @@
             inherit cargoArtifacts;
             buildPhase = ''
               # Run the tests for Charon
-              DEST=$out CHARON="${charon}/bin/charon --cargo-no-rust-version" \
+              DEST=$out CARGO_NO_RUST_VERSION=1 CHARON_DRIVER="${charon}/bin/charon-driver" \
```

Comment:
Do we agree that with your changes `charon` runs cargo while `charon-driver`doesn't (and in particular `charon` is still valid)?

Reply:
Yes, `charon` is unchanged. We could probably rename it to `cargo-charon` tbh